client/reader/components/icons/a8c-icon.jsx

```diff
@@ -0,0 +1,20 @@
+export default function ReaderA8cIcon() {
+	return (
+		<svg
+			className="sidebar__menu-icon sidebar_svg sidebar_svg-a8c"
+			fill="none"
+			height="20"
+			viewBox="0 0 20 20"
+			width="20"
+			xmlns="http://www.w3.org/2000/svg"
+		>
+			<g stroke="#a2aab2" strokeLinecap="round" strokeWidth="1.5">
```

Comment:
Same as above with regards to applying colors and other color scheming styling to icons. Applicable to all SVG components in this PR actually.